docs/.eleventy.js

```diff
@@ -123,19 +123,27 @@ module.exports = function(eleventyConfig) {
     /**********************************************************************
      *  Shortcodes
      * ********************************************************************/
-    eleventyConfig.addNunjucksAsyncShortcode("link", async function(link) {
-        const { body: html, url } = await got(link);
-        const metadata = await metascraper({ html, url });
-        const the_url = (new URL(link)); // same as url
-        const domain = the_url.hostname;
+    eleventyConfig.addNunjucksShortcode("link", function(url) {
+
+        const urlData = this.ctx.further_reading_links[url];
+
+        if (!urlData) {
+            throw new Error(`Data missing for ${url}`);
+        }
+
+        const {
+            domain,
+            title,
+            logo
+        } = urlData;
 
         return `
         <article class="resource">
             <div class="resource__image">
-                <img class="resource__img" width="75" height="75" src="${metadata.logo}" alt="Avatar image for ${domain}" />
+                <img class="resource__img" width="75" height="75" src="${logo}" alt="Avatar image for ${domain}" onerror="this.src = '/icon.svg'" />
```

Comment:
```suggestion
                <img class="resource__img" width="75" height="75" src="${logo}" alt="Avatar image for ${domain}" onerror="this.onerror = null; this.src = '/icon.svg';" />
```

To avoid an endless loop when there happens to be something wrong with loading the fallback image.